packages/terra-list/src/ListSection.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
this should not be added for whole file instead you can add it above the statement that causing lint error.

Reply:
Changes done. Thanks